test/shared/box.lua

```diff
@@ -134,6 +134,28 @@ box.once('initialization', function()
     local tuple = yaml.decode(yml)[1]
     tuple[1] = "12345"
     box.space._schema:insert(tuple)
+
+    --gh-151 Support new _index system space format
+    -- After tarantool-1.7.5-153-g1651fc9be the new _index format
+    -- was introduced. When an index part uses is_nullable or
+    -- collation parameter, then the new format will be used.
+    if tarantool_version_at_least(1, 7, 6, 0) then
+        local test_new_format = box.schema.space.create('test_new_format', {
```

Comment:
We want to test new storage format of index parts, but 'format' is another thing in tarantool. The name is a bit confusing.

Reply:
I haven't been able to come up with a good self-documenting name. I agree with any suggestion you make.

Reply:
`test_index_part_176`?

Reply:
Ok. Updated.